glib-macros/tests/test.rs

```diff
@@ -652,4 +652,48 @@ fn closure() {
         };
         cast_test.invoke::<()>(&[]);
     }
+
+    {
```

Comment:
Why is this test in glib-macros?

Reply:
No particular reason, all the proc macro tests were there before